regulations/uitests/definition_test.py

```diff
@@ -77,20 +78,19 @@ def test_definition(self):
         # go to 1005-1-a
         toc_toggle.click()
         WebDriverWait(self.driver, 10)
-        self.driver.execute_script(
-            "window.scrollTo(0, document.body.scrollHeight);")
+        self.driver.execute_script('window.scrollTo(0, 5);')
```

Comment:
This is a different test, hey? Previously it was scrolling to the bottom of the doc?